src/sql/platform/dialog/wizardModal.ts

```diff
@@ -53,6 +56,7 @@ export class WizardModal extends Modal {
 		@IClipboardService clipboardService: IClipboardService
 	) {
 		super(_wizard.title, name, partService, telemetryService, clipboardService, themeService, contextKeyService, options);
+		this._useDefaultMessageBoxLocation = false;
```

Comment:
Would it make sense to have this as part of the options argument that gets passed to the constructor?

Reply:
I did this on purpose, trying to make things simple for the extension builder. I don't see a need for this to be configurable for wizard at this moment, we can make it an option in the future when we really need to.